docs/static/style-guide/formats.md

```diff
@@ -0,0 +1,197 @@
+    Example: sszvis.formatNumber(16000)
+    Result: "16 000"
+  - Function:
+    Example: sszvis.formatNumber(5000)
+    Result: "5000"
+```
+
+#### `sszvis.formatPreciseNumber(p, d)`
+
+`p` = The desired precision
+`d` = The number to be formatted
+
+A number will be formatted to a particular precision. It is a function that accepts numbers and returns formatted values. This function is "curried", meaning that it is a function with multiple arguments, but when you call it with less than the full number of arguments, it returns a function that takes less arguments and has the arguments you did provide "pre-filled" as parameters.
+
+Note that preciseNumber(2, 14.1234) is equivalent to preciseNumber(2)(14.1234)
+
+```table
+rows:
+  - Function: formatPreciseNumber
+    Example: sszvis.formatPreciseNumber(2, 14.1234)
+    Result: 14.12
+  - Function:
+    Example: sszvis.formatPreciseNumber(3, 14.1234)
+    Result: 14.123
+  - Function:
+    Example: sszvis.formatPreciseNumber(3)(14.1234)
+    Result: 14.123
+  - Function:
+    Example: sszvis.formatPreciseNumber(0, 14.1234)
+    Result: 14
+```
+
+#### `sszvis.formatPercent(x)`
+
+A number will be transformed into a percent format. Formats percentages on the range 0 - 100.
+
+```table
+rows:
+  - Function: formatPercent
+    Example: sszvis.formatPercent(55.3)
+    Result: "55.3 %"
+  - Function:
+    Example: sszvis.formatPercent(7)
+    Result: "7 %"
+  - Function:
+    Example: sszvis.formatPercent(0.245875)
+    Result: "0.25 %"
+```
+
+#### `sszvis.formatFractionPercent(x)`
+
+A number will be transformed into a percent format. Format percentages on the range 0 - 1.
+
+```table
+rows:
+  - Function: formatFractionPercent
+    Example: sszvis.formatFractionPercent(55.3)
+    Result: "5530 %"
+  - Function:
+    Example: sszvis.formatFractionPercent(7)
+    Result: "700 %"
+  - Function:
+    Example: sszvis.formatFractionPercent(0.245875)
+    Result: "24.59 %"
+```
+
+#### `sszvis.formatAge(x)`
+
+A number will be formatted as an age. Means that any decimal number will be rounded to a full number.
+
+```table
+rows:
+  - Function: formatAge
+    Example: sszvis.formatAge(55.54359375)
+    Result: "56"
+  - Function:
+    Example: sszvis.formatAge(555435)
+    Result: "555435"
+```
+
+#### `sszvis.formatYear(x)`
+
+A date object will be formatted as a year. Gives the date's year.
+
+```table
+rows:
+  - Function: formatYear
+    Example: sszvis.formatYear(new Date())
+    Result: "2020"
+```
+
+#### `sszvis.formatMonth(x)`
+
+A date object will be formatted as a month. It will return a capitalized three-letter abbreviation of the German month name.
+
+```table
+rows:
+  - Function: formatMonth
+    Example: sszvis.formatMonth(new Date())
+    Result: "JAN"
+```
+
+#### `sszvis.formatAxisTimeFormat(x)`
+
+A multi time formatter used by the axis class.
+
+```table
+rows:
+  - Function: formatAxisTimeFormat
+    Example: sszvis.formatAxisTimeFormat(new Date())
+    Result: ".345"
```

Comment:
This is an important function, but this example doesn't show what it does very well. Maybe we should not show an example and just say that it is used by d3 under the hood. (See https://github.com/d3/d3-time-format)

Reply:
Good input, but how do we explain when it can be used?